### PR TITLE
[ADD] 9.0 - Added purchase order report in qweb with notes 

### PR DESCRIPTION
**Pull request by SerpentCS**

### CODE REVIEW

**Comment by AaronHForgeFlow** [comment]
Can you please add some description?

File: purchase_order_report_with_note/README.rst [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+Purchase Order Report With Note
+===============================
+
+* This module replaces the webkit Quotation / Purchase Order report by
+  a qweb report and adds header and footer notes on the purchase order 
+  and quotation
+
+
+Installation
+============
```
Comment (AaronHForgeFlow) [inline purchase_order_report_with_note/README.rst]: Please remove empty sections

File: purchase_order_report_with_note/README.rst [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+No specific installation requirements.
+
+
+Configuration
+=============
```
Comment (AaronHForgeFlow) [inline purchase_order_report_with_note/README.rst]: Here also

File: purchase_order_report_with_note/__init__.py [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# -*- coding: utf-8 -*-
+#   Copyright (c) 2011 Camptocamp SA (http://www.camptocamp.com)
+#   @author Guewen Baconnier
```
Comment (AaronHForgeFlow) [inline purchase_order_report_with_note/__init__.py:3]: I'd remove this from init files. https://github.com/OCA/maintainer-tools/pull/260 But it's not blocking anyway

**Review by AaronHForgeFlow — COMMENTED** [review]
It works fine. Just some comments

**Comment by pedrobaeza** [comment]
@SerpentCS are you interested in finish this?

**Comment by SerpentCS** [comment]
@chandresh-serpentcs will check

**Review by dreispt — APPROVED** [review]

**Pull request closed without merging**